client/state/data-layer/wpcom/read/following/mine/delete/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External Dependencies
+ */
+import { translate } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import config from 'config';
+import { READER_UNFOLLOW } from 'state/action-types';
+import { dispatchRequest } from 'state/data-layer/wpcom-http/utils';
+import { http } from 'state/data-layer/wpcom-http/actions';
+import { errorNotice } from 'state/notices/actions';
+import { follow } from 'state/reader/follows/actions';
+
+export function requestUnfollow( { dispatch, getState }, action, next ) {
+	const { payload: { feedUrl } } = action;
+	dispatch( http( {
+		method: 'POST',
+		path: '/read/following/mine/delete',
+		apiVersion: '1.1',
+		body: {
+			url: feedUrl,
+			source: config( 'readerFollowingSource' )
+		},
+		onSuccess: action,
+		onFailure: action,
+	} ) );
+	next( action );
+}
+
+export function receiveUnfollow( store, action, next, response ) {
+	if ( response && ! response.subscribed ) {
```

Comment:
just realizing now that we should have made `dispatchRequest` take in named params instead